### PR TITLE
Now Support Heroku Via Manual Deploy I Will add Heroku Button later

### PR DESCRIPTION
**Pull request by sppidy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,10 +24,20 @@ Please be reminded that we're NOT responsible for your TOS violations.
 
 ### With Heroku
 
-Currently, not yet supported. It will, very soon.
+**Note :- The Support To Heroku Vars Will be Given Soon**
```
Comment (robbyoconnor) [inline README.md:27]: What?

File: README.md [hunk 2]
```diff
@@ -22,10 +24,20 @@ Please be reminded that we're NOT responsible for your TOS violations.
...
+
+**Do The Following Steps to Deploy Via Heroku**
+
+1. Fork The Repo
+2. Copy the sample config file with `cp spam/config.sample.py` And Create a New FIle Named `spam/config.py` Paste The Sample Config and edit it.
```
Comment (robbyoconnor) [inline README.md:32]: `cp spam/config.sample.py spam/config.py` -- There is no need to copy/paste.

File: README.md [hunk 3]
```diff
@@ -22,10 +24,20 @@ Please be reminded that we're NOT responsible for your TOS violations.
...
+
+Then Watch the Below Video to Deploy Manually :
+
+https://www.youtube.com/watch?v=d8Kxt717W0k
+
```
Comment (robbyoconnor) [inline README.md:37]: I don't see a point with the video. Just give the commands, they're simple.

File: spam/config.sample.py [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
+class Config:
+        # To run the userbot, make this variable defined. DO NOT LEAVE IT BLANK!
+        ENABLE = ""
 
-# Need to debug stuff? Enable this one to show debug logs which might contains
-# senstive information.
-LOGGER = ""
+        # Need to debug stuff? Enable this one to show debug logs which might contains
+        # senstive information.
+        LOGGER = ""
```
Comment (robbyoconnor) [inline spam/config.sample.py:10]: Nuke the indentation.

File: spam/config.sample.py [hunk 5]
```diff
@@ -1,18 +1,19 @@
...
 
-# Command prefix for userbot commands.
-Command = "!"
+        # Command prefix for userbot commands.
+        Command = "!"
```
Comment (robbyoconnor) [inline spam/config.sample.py:13]: Nuke the indentation.

File: spam/config.sample.py [hunk 6]
```diff
@@ -1,18 +1,19 @@
...
-# Get one from https://my.telegram.org/apps
-user1_api_id = ""
-user1_api_hash = ""
-user1_session_string = ""
\ No newline at end of file
+        # ID and Hash for first user.
+        # Get one from https://my.telegram.org/apps
+        user1_api_id = ""
+        user1_api_hash = ""
+        user1_session_string = ""
```
Comment (robbyoconnor) [inline spam/config.sample.py:19]: Nuke the indentation.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
I would strongly recommend against merging this @Netkas -- it's pretty bad.

File: README.md (hunk 1 above)
Comment (robbyoconnor) [reply README.md:27]: Use Environment variables. We should be doing that anyway.

**Comment by AntiEngineer** [comment]
Hello.

I'm closing this pull request because the following reasons:
  - The code quality isn't optimal and it's not what we are looking for.
  - Heroku is a cloud-testing, CI, (and more) platform. This project on Heroku violates their acceptable use policy you can find at https://www.heroku.com/policy/aup.

This decision is not reversable and won't be analyzed by our staff. 
#2917-100173003-30IVA-BLOCK-1928

**Pull request closed without merging**